src/Public.Api/TicketingService/TicketingServiceController.cs

```diff
@@ -0,0 +1,38 @@
+namespace Public.Api.TicketingService
+{
+    using Autofac.Features.AttributeFilters;
+    using Be.Vlaanderen.Basisregisters.Api;
+    using Common.Infrastructure;
+    using Common.Infrastructure.Controllers;
+    using Common.Infrastructure.Controllers.Attributes;
+    using FeatureToggle;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+    using Infrastructure.Configuration;
+    using Infrastructure.Swagger;
+    using RestSharp;
+    using Version = Infrastructure.Version.Version;
+
+    [ApiVisible]
+    [ApiVersion(Version.Current)]
```

Comment:
Do we want the current version, or maybe across all versions? using `[ApiVersionNeutral]`.
If we do want to version it, I would recommend starting from `V2` so it aligns with our edit api's.

Reply:
We want to version everything, so I set it to V2.